package.json

```diff
@@ -8,10 +8,110 @@
+    "babel-plugin-transform-es2015-object-super": "^6.22.0",
+    "babel-plugin-transform-es2015-parameters": "^6.23.0",
+    "babel-plugin-transform-es2015-shorthand-properties": "^6.22.0",
+    "babel-plugin-transform-es2015-spread": "^6.22.0",
+    "babel-plugin-transform-es2015-sticky-regex": "^6.22.0",
+    "babel-plugin-transform-es2015-template-literals": "^6.22.0",
+    "babel-plugin-transform-es2015-typeof-symbol": "^6.23.0",
+    "babel-plugin-transform-es2015-unicode-regex": "^6.22.0",
+    "babel-plugin-transform-es3-member-expression-literals": "^6.22.0",
+    "babel-plugin-transform-es3-property-literals": "^6.22.0",
+    "babel-plugin-transform-es5-property-mutators": "^6.22.0",
+    "babel-plugin-transform-eval": "^6.22.0",
+    "babel-plugin-transform-exponentiation-operator": "^6.22.0",
+    "babel-plugin-transform-export-extensions": "^6.22.0",
+    "babel-plugin-transform-flow-comments": "^6.22.0",
+    "babel-plugin-transform-flow-strip-types": "^6.22.0",
+    "babel-plugin-transform-function-bind": "^6.22.0",
+    "babel-plugin-transform-inline-environment-variables": "^6.8.0",
+    "babel-plugin-transform-jscript": "^6.22.0",
+    "babel-plugin-transform-member-expression-literals": "^6.8.1",
+    "babel-plugin-transform-merge-sibling-variables": "^6.8.2",
+    "babel-plugin-transform-minify-booleans": "^6.8.0",
+    "babel-plugin-transform-node-env-inline": "^6.8.0",
+    "babel-plugin-transform-object-assign": "^6.22.0",
+    "babel-plugin-transform-object-rest-spread": "^6.23.0",
+    "babel-plugin-transform-object-set-prototype-of-to-assign": "^6.22.0",
+    "babel-plugin-transform-property-literals": "^6.8.1",
+    "babel-plugin-transform-proto-to-assign": "^6.23.0",
+    "babel-plugin-transform-react-constant-elements": "^6.23.0",
+    "babel-plugin-transform-react-display-name": "^6.23.0",
+    "babel-plugin-transform-react-inline-elements": "^6.22.0",
+    "babel-plugin-transform-react-jsx": "^6.23.0",
+    "babel-plugin-transform-react-jsx-compat": "^6.22.0",
+    "babel-plugin-transform-react-jsx-self": "^6.22.0",
+    "babel-plugin-transform-react-jsx-source": "^6.22.0",
+    "babel-plugin-transform-regenerator": "^6.22.0",
+    "babel-plugin-transform-remove-console": "^6.8.0",
+    "babel-plugin-transform-remove-debugger": "^6.8.0",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-plugin-transform-simplify-comparison-operators": "^6.8.1",
+    "babel-plugin-transform-strict-mode": "^6.22.0",
+    "babel-plugin-transform-undefined-to-void": "^6.8.0",
+    "babel-plugin-undeclared-variables-check": "^6.22.0",
+    "babel-preset-babili": "^0.0.11",
+    "babel-preset-es2015": "^6.22.0",
+    "babel-preset-es2016": "^6.22.0",
+    "babel-preset-es2017": "^6.22.0",
+    "babel-preset-latest": "^6.22.0",
+    "babel-preset-react": "^6.23.0",
+    "babel-preset-stage-0": "^6.22.0",
+    "babel-preset-stage-1": "^6.22.0",
+    "babel-preset-stage-2": "^6.22.0",
+    "babel-preset-stage-3": "^6.22.0",
```

Comment:
why are all these needed?  How will this affect DoneJS's and steal-tool's load times?